packages/attention/src/component.tsx

```diff
@@ -91,47 +100,52 @@ export function Attention(props: AttentionProps) {
       )}
       ref={attentionRef}
     >
-      <div
-        className={classNames({
-          [c.base]: true,
-          [c.tooltip]: props.tooltip,
-          [c.callout]: props.callout,
-          [c.popover]: props.popover,
-        })}
-      >
+      <div className={wrapperClasses}>
         {!props.noArrow && (
           <Arrow {...props} ref={arrowRef} direction={placement} />
         )}
-        <div className="last-child:mb-0">{props.children}</div>
+        <div className="last:mb-0">{props.children}</div>
       </div>
     </div>
   );
 }
 
-const Arrow = forwardRef<HTMLDivElement, ArrowProps>((props, ref) => {
-  const { direction, tooltip, callout, popover } = props;
+const arrowDirectionClassname = (dir: string) => {
+  const direction = dir.charAt(0).toUpperCase() + dir.slice(1);
+
+  return `arrowDirection${direction}`;
+}
 
+const Arrow = forwardRef<HTMLDivElement, ArrowProps>((props, ref) => {
+  const { callout, direction, popover, tooltip } = props;
   const arrowDirection = opposites[direction];
 
+  const arrowType = () => {
+    if (tooltip) return 'arrowTooltip';
+    else if (callout) return 'arrowCallout';
+    else if (popover) return 'arrowPopover';
+    return '';
+  }
+
+  const arrowClasses = classNames(
+    ccAttention.arrowBase,
+    ccAttention[arrowDirectionClassname(arrowDirection)],
+    {
+      [ccAttention[arrowType()]]: tooltip || callout || popover
+    },
+  );
+
   return (
     <div
       aria-label={arrowLabels[arrowDirection]}
       ref={ref}
-      className={classNames({
-        [c.arrowBase]: true,
-        [`-${arrowDirection}-8`]: true,
-        [c.arrowTooltip]: tooltip,
-        [c.arrowCallout]: callout,
-        [c.arrowPopover]: popover,
-      })}
+      className={arrowClasses}
       style={{
         // TW doesn't let us specify exactly one corner, only whole sides
         borderTopLeftRadius: '4px',
         zIndex: 1,
         // border alignment is off by a fraction of a pixel, this fixes it
-        [`margin${
-          arrowDirection.charAt(0).toUpperCase() + arrowDirection.slice(1)
-        }`]: '-0.5px',
+        [`margin${arrowDirection.charAt(0).toUpperCase() + arrowDirection.slice(1)}`]: '-0.5px',
```

Comment:
Can `arrowDirectionClassname ` be used here? The logic looks the same but I haven't checked more specifically